### ISSUE TITLE
Add integration tests for suspicious activity interface

### ISSUE THREAD

**Issue by cssru** [php-coder/mystamps#362]
Insert suspicious activity data into database.
Request suspicious activities page.
Check that record about suspicious activity is present on generated web-page.
Tech debt for: b15bea1 (#248)

**Comment by php-coder** [comment]
Related to other issues for suspicious activity: #350, #255, #273

**Comment by php-coder** [comment]
I don't think that we should modify database directly, instead we should simulate the situation and produce such activity.

Also we should check for a presence of the link to page with events on index page.
